zetasql/parser/bison_parser.y

```diff
@@ -3375,27 +3382,45 @@ export_model_statement:
     ;
 
 grant_statement:
-    "GRANT" privileges "ON" identifier path_expression "TO" grantee_list
+    "GRANT" privileges "ON" grant_object_kind grant_path "TO" grantee_list opt_grant_option
```

Comment:
grant_object_kind should be identifier only

Reply:
identifier cannot cover two keywords

Reply:
it's target_type, never being two words